index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
```

Comment:
Great job, you added `indexl.html` and `sytle.css` but for some reason, you forgot to add `test.md`.
According to the project requirement you're supposed to add `test.md`, kindly add it =.